### PR TITLE
[Focused improvement] Add a linting gate for git push actions

### PR DESCRIPTION
**Pull request by CobyPear**
Add a script that 'installs' a pre-commit hook which will lint packages and apps on ~~commit~~ push. If you'd like to skip this hook, add the `--no-verify` flag to your ~~commit~~ push i.e. `git push origin head --no-verify`
## Description / Motivation

Instead of waiting for CI to potentially fail due to unlinted apps/packages, we want to catch those errors locally.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update (non-breaking change; modified files are limited to the `/docs` directory)

### CODE REVIEW

File: scripts/installHooks.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const fs = require('fs');
```
Comment (ambrauer) [inline scripts/installHooks.js:1]: Use our newer file naming convention here "install-hooks.js"?

**Review by ambrauer — APPROVED** [review]
Just the one minor suggestion, otherwise this looks good to me. Certain git GUI may not easily facilitate the --no-verify flag, so I like that this is opt-in.

File: scripts/installHooks.js (hunk 1 above)
Comment (CobyPear) [reply scripts/installHooks.js]: Good call!

**Review by ambrauer — COMMENTED** [review]
One other thought. Can/should we make this a pre-push hook instead of a pre-commit? Would still solve our main goal of preventing linting issues from making it to origin, but less obtrusively to dev workflow.

**Comment by CobyPear** [comment]
> One other thought. Can/should we make this a pre-push hook instead of a pre-commit? Would still solve our main goal of preventing linting issues from making it to origin, but less obtrusively to dev workflow.

Yeah absolutely. Will change!

**Pull request merged**